### PR TITLE
Add saucelabs configuration and execution

### PR DESCRIPTION
**Pull request by germandavid85**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+  - secure: bTU9G+Iv3ZeIZ8bKEI3qGHUisX6MbFVZtpBusEWBZRw7Tw8L0gtFLhfUKwdChlhkblSKvSM2U+Mrk53Gb7FdK9yrHiyTSxHX563jvsZNrQIPWwbBo/jU+Z5IWQ/ZNKF1lh+uOVZHiufg6Xa+vDb5N113Yma+KSZDvreD7wB9rrVZG3lX6MrGdbu8diw9H7sMdYQERpNbU61v8Uj6PEHu+XQMJ/jlFVbzjzb4l8X9jufMgGVdVnMqhTaqY3aNLfm7WBJMoMF4XLLm1MMsLVlhsKJbLwpH5UHy6NTtgNGcn0ccZ4T3xHKXalB4fahrBHnTy89Q+7esE3AVUkLPKPgrB5DeN4fmKGVF8PHfh5z0/nRrv6Mqv3aA9Tist8HA0JLckDURoGoEZ80MhE7Xc624CpvFIrgM59Oo48LCnc8ibzOhonTs5gCWq79UbwAk5Z0y5NybE/Zg0hbqaiEivqbw8vlfsgjc4Mkz9F/nx3R0zmYLDzPySdabQHIqngLWgtFu51EuoZFdz8Uhc0ukQX4XWc/HOqPyyvxMBAPWBy2paQu+QMHFVYKtxlcq8d0lX7NxZ791hIKwnHGDZCsbZ1Mxad5qMkdbCVKVu/xGOT0qg8rcD9PuAbJXBOnRnsyr2gCLzHMLQnecUIGQO7v7TRy+UPvTTtRdUpMCEV+7Zc92fxg=
```
Comment (germandavid85) [inline .travis.yml]: this is just for sample with my credentials, but we can make register in saucelabs as an opensource project and have a less restricted account

Comment (aperdomob) [reply .travis.yml]: Podriamos crear una cuenta para proyecto opensource?

Comment (jhenaoz) [reply .travis.yml]: si

File: protractor/saucelabs.config.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { browser, Config } from 'protractor';
+import { reporter } from './helpers/reporter';
+
+export let config: Config = {
+  framework: 'jasmine',
+  specs: ['../test/**/*.spec.js'],
+  SELENIUM_PROMISE_MANAGER: false,
```
Comment (jhenaoz) [inline protractor/saucelabs.config.ts]: vamos a usar async await? o porque ponemos esto en falso?

Comment (aperdomob) [reply protractor/saucelabs.config.ts]: papí esto esta prácticamente desde el tercer commit
si vamos a utilizar async/await ois

**Pull request merged**